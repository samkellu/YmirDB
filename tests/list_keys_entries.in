list entries
list keys
set a 123 123 22 33
set b a 22 11 44
set c a b 22 11
list entries
list keys
bye
