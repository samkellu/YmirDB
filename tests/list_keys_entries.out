> no entries

> no keys

> ok

> ok

> ok

> c [a b 22 11]
b [a 22 11 44]
a [123 123 22 33]

> c
b
a

> bye
